optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -750,9 +751,9 @@ class LightGBMTuner(_LightGBMBaseTuner):
     def __init__(
         self,
         params: dict[str, Any],
-        train_set: "lgb.Dataset",
+        train_set: lgb.Dataset,
         num_boost_round: int = 1000,
-        valid_sets: "VALID_SET_TYPE" | None = None,
+        valid_sets: list[lgb.Dataset] | tuple[lgb.Dataset, ...] | lgb.Dataset | None = None,
```

Comment:
Also, please stringify this type annotation.
```suggestion
        valid_sets: "list[lgb.Dataset] | tuple[lgb.Dataset, ...] | lgb.Dataset | None" = None,
```